Automattic/jetpack
# Try: Only attempt to make videos responsive if MediaElement present in WordPress is not doing it already

Fixes 8215 for Jetpack sites

#### Changes proposed in this Pull Request:

* Updates `responsive-videos.js` to do nothing if `MediaElement.js` present in core is already making media be responsive.

#### Testing instructions:

* Start with a connected Jetpack site running on WordPress 4.9.
  * Checkout this branch.
  * Activate theme Twenty Sixteen on the site.
  * Add a Video Widget with a Vimeo video.
  * Add a Video Widget with a Youtube video.
  * Enable the constant `SCRIPT_DEBUG` so you don't get the minified version of the plugin
  * On the customizer add this CSS:
  ```
  .site-inner {
    max-width: 5000px ;
  }
  ```
* With Chrome Dev Tools, try to emulate a device with resolution like `4500x1800`.
* Confirm that both videos play well without going into fullscreen. Audio and video should be perceived.
* Confirm that going fullscreen fills the whole screen and works well.
